packages/flutter_test/test/widget_tester_live_device_test.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+
+void main() {
+  final _MockLiveTestWidgetsFlutterBinding binding = _MockLiveTestWidgetsFlutterBinding();
+
+  testWidgets('Should print message on pointer events', (WidgetTester tester) async {
+    final List<String?> printedMessages = <String?>[];
+
+    int invocations = 0;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Center(
+          child: GestureDetector(
+            onTap: () {
+              invocations++;
+            },
+            child: const Text('Test'),
+          ),
+        ),
+      ),
+    );
+
+    final Size windowCenter = tester.binding.window.physicalSize /
+        tester.binding.window.devicePixelRatio /
+        2;
+    final double windowCenterX = windowCenter.width;
+    final double windowCenterY = windowCenter.height;
+
+    final Offset widgetCenter = tester.getRect(find.byType(Text)).center;
+    expect(widgetCenter.dx, windowCenterX);
+    expect(widgetCenter.dy, windowCenterY);
+
+    await binding.collectDebugPrints(printedMessages, () async {
+      await tester.tap(find.byType(Text));
+    });
+    await tester.pump();
+    expect(invocations, 0);
+
+    expect(printedMessages, equals('''
```

Comment:
Maybe to verify that it doesn't just list all possibilities also check what it prints when the tap misses the Text widget?